src/InsertionsClient.Core/Api/Props/Models/PropsFile.cs

```diff
@@ -46,6 +47,16 @@ internal PropsFile(string path)
 
         private bool _isLoaded => _xDocument != null;
 
+        /// <summary>
+        /// Settings to make sure saved xml has correct the formatting.
+        /// </summary>
+        private static readonly XmlWriterSettings _xmlWriteSettings = new XmlWriterSettings()
+        {
+            OmitXmlDeclaration = true,
+            Indent = true,
+            Encoding = Encoding.ASCII
```

Comment:
Looks like you're changing the encoding from the default UTF-8 to ASCII. Is there a reason for that?

Reply:
Default.config and props files on VS are already using ASCII encoding. I also suspect that the tooling we have around these files may not have UTF-8 support. This should be safer and coherent with the existing files.

Reply:
Thank you for the explanation. :shipit: